selftests/run_coverage

```diff
@@ -6,7 +6,9 @@
 # Author: Lukas Doktor <ldoktor@redhat.com>
 
 coverage erase
-coverage run --include "avocado/*" ./selftests/run
+coverage rm .coverage.*
```

Comment:
~~~
$ coverage rm .coverage.*
Unknown command: 'rm'
Use 'coverage help' for help.
~~~